### ISSUE TITLE
The Donation link that se send in email is invalid

### ISSUE THREAD

**Issue by mohammadranjbarz** [Giveth/giveth-next#652]
![Screen Shot 1400-10-30 at 22.54.39.png](https://images.zenhubusercontent.com/5fa7ed7fb0994323d1a66034/162fbbc7-a0eb-4005-a9a9-870332677ce9)

@MoeNick  @divine-comedian  
I have donated to a project, and in email that I received was a button to `View Your Donations`, and when I clicked on that, it redirected me to an invalid link

https://giveth.apms5.com/anywhere/m?s=giveth&m=tr_3697f8ad-89b5-4a1d-be76-bee407959f60&u=e1jq4wvfdtfk2gu16ctkeh9p5mtm2c235mu3jg9h5n146chn5n0k2h9h64t38d1t6rwke&r2=d1u78w3k78qjytv9etjq8u1ed5qjyrb3cdqqavkm7xv6jtbq7nj6yvk1ehmpyvkk&n=4

**Comment by divine-comedian** [comment]
can you provide more information? what browser are you using? I have received the same email and the link properly redirects me to `my donations` page on giveth.io - the link you posted in this issue also redirects me to `my donations`

**Comment by mohammadranjbarz** [comment]
> can you provide more information? what browser are you using? I have received the same email and the link properly redirects me to `my donations` page on giveth.io - the link you posted in this issue also redirects me to `my donations`

I donated with Google chrome in production (giveth.io)
I will forward the email that I received to you

**Comment by mohammadranjbarz** [comment]
> can you provide more information? what browser are you using? I have received the same email and the link properly redirects me to `my donations` page on giveth.io - the link you posted in this issue also redirects me to `my donations`

You are right, it was my internet problem, now it redirects me to my donations page
